.github/workflows/kotlin.yml

```diff
@@ -470,7 +470,28 @@ jobs:
         uses: mikepenz/action-junit-report@5f47764eec0e1c1f19f40c8e60a5ba47e47015c5 # v4
         if: always() # always run even if the previous step fails
         with:
-          report_paths: '**/build/test-results/test/TEST-*.xml'
+          report_paths: '**/build/test-results/jvmTest/TEST-*.xml'
+
+  jvm-all-runtime-test:
+    name: ALL Optimizations JVM Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Check with Gradle
+        uses: ./.github/actions/gradle-task
+        with:
+          task: jvmTest --continue -Pworkflow.runtime=all
```

Comment:
I don't test the work stealing dispatcher combinations separately, just adding the 'all' config here for the jvm tests.

Reply:
Which should be fine b/c we plan to ship that way?